README.md

```diff
@@ -15,27 +15,34 @@ compliant with HTML5 and they will look OK.
 
 Details are here: [yegor256.github.io/tacit](http://yegor256.github.io/tacit/)
 
-Just add it to your HTML (replace `VERSION` with the latest version number,
-which you can find [here](https://github.com/yegor256/tacit/releases)):
+Start with this basic HTML template and modify it to suit your needs
 
 ```html
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <link rel="stylesheet" href="https://cdn.rawgit.com/yegor256/tacit/gh-pages/tacit-VERSION.min.css"/>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://cdn.rawgit.com/yegor256/tacit/gh-pages/tacit-0.8.1.min.css"/>
   </head>
+  <body>
+    <!-- Webpage content goes here -->
+  </body>
 </html>
 ```
 
-Or simply download [`tacit.min.css`](https://raw.githubusercontent.com/yegor256/tacit/gh-pages/tacit.min.css)
+Or simply download [`tacit.min.css`](https://cdn.rawgit.com/yegor256/tacit/gh-pages/tacit-0.8.1.min.css)
```

Comment:
@smartclash please, remove this change. We should point to the latest version always, in the `README`.